app/src/androidTest/java/org/mozilla/fenix/ui/ContextMenusTest.kt

```diff
@@ -129,13 +129,12 @@ class ContextMenusTest {
     }
 
     @Test
-    fun verifyContextCopyLinkNotDisplayedAfterApplied() {
+    fun verifyLinkFromClipboardNotDisplayedTest() {
```

Comment:
I think the former test title was more descriptive.

Reply:
Done !